.github/workflows/run-end-to-end.yml

```diff
@@ -179,12 +179,13 @@ jobs:
     - name: Load library binary
       if: ${{ matrix.version == 'dev' }}
       run: ./utils/scripts/load-binary.sh ${{ inputs.library }}
-    
       # quick and dirty fix, because we can't test for now the PHP released tracer
     - name: Load PHP library binary (skipping release tests, use snapshot instead)
       if: ${{ matrix.version == 'prod' && inputs.library == 'php'}}
       run: ./utils/scripts/load-binary.sh ${{ inputs.library }}
-
+    - name: Load Ruby library binary (skipping release tests, use snapshot instead)
```

Comment:
You can remove this line, as you are using the pre-release gem, which is already named `datadog`

Reply:
I have tried, the test failed https://github.com/DataDog/system-tests/pull/2222/commits/5859ac5e56dccb269e9db081f04d4cdd517408b1